server/src/services/order.service.ts

```diff
@@ -1,12 +1,38 @@
 import { Op } from 'sequelize';
 import { Transaction } from 'sequelize/types';
-import { ICreateOrderRequest } from '../interfaces/orderRequest.interface';
+import { ICreateOrderRequest, orderStateType } from '../interfaces/orderRequest.interface';
 import Item from '../models/item.model';
+import ItemOption from '../models/itemOption.model';
 import Order from '../models/order.model';
 import OrderAddress from '../models/orderAddress.model';
 import OrderItem from '../models/orderItem.model';
 
 class OrderService {
+  public async getAll(user_id: string, state: orderStateType, start: Date, end: Date): Promise<Order[]> {
```

Comment:
```suggestion
  public async getAll(user_id: string, state: OrderStateType, start: Date, end: Date): Promise<Order[]> {
```

변수와 혼동되지 않게 UpperCamelCase를 쓰는 건 어떨까요?

Reply:
수정했습니다!!